drivers/bridge/setup_ip_tables.go

```diff
@@ -13,6 +13,7 @@ import (
 const (
 	DockerChain    = "DOCKER"
 	IsolationChain = "DOCKER-ISOLATION"
+	fwdTopChain    = "DOCKER-FORWARD-TOP"
```

Comment:
+1.

In service_linux.go:
  // In the filter table FORWARD chain first rule should be to jump to INGRESS-CHAIN